Modix/Modules/NoteModule.cs

```diff
@@ -0,0 +1,172 @@
+﻿namespace Modix.Modules
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Discord;
+    using Discord.Commands;
+    using Discord.WebSocket;
+    using Modix.Data.Models;
+    using Modix.Data.Models.Moderation;
+    using Modix.Services.Moderation;
+    using Serilog;
+    using Services.CommandHelp;
+    using Tababular;
+
+    [Name("Add/Remove/Search Admin notes"), RequireUserPermission(GuildPermission.Administrator), HiddenFromHelp]
+    public class NoteModule : ModuleBase
```

Comment:
This is going to make the command only work for server admins.

Reply:
Fixed in cd217ce.